src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -37,6 +37,7 @@ static ErrorFacts()
             builder.Add(getId(ErrorCode.WRN_NullabilityMismatchInTypeParameterReferenceTypeConstraint));
             builder.Add(getId(ErrorCode.WRN_CantInferNullabilityOfMethodTypeArgs));
             builder.Add(getId(ErrorCode.WRN_NoBestNullabilityArrayElements));
+            builder.Add(getId(ErrorCode.WRN_PossibleNull));
```

Comment:
Doesn't need to be addressed in this PR, but do we have any way of verifying that all nullability warnings are getting added to this builder? This seems like a process that will be prone to breaking in the future.